iceoryx_hoofs/utility/include/iox/detail/convert.inl

```diff
@@ -363,16 +363,20 @@ inline bool convert::is_within_range(const SourceType& source_val) noexcept
     {
         return true;
     }
-    // is_arithmetic_v
     if constexpr (std::is_floating_point_v<SourceType>)
     {
         // special cases for floating point
         if (std::isnan(source_val) || std::isinf(source_val))
         {
             return true;
         }
+#ifdef _MSC_VER
```

Comment:
Is this compiler switch required? If yes, then it needs to be `_WIN32` since I noticed a similar behavior with MinGW where i had to temporarily deactivate some tests on the branch in order to make the CI happy.

Reply:
Oh, you're correct. I'll remove the branch cause it seems ok to apply to every circumstance.